test/functional/apm.test.js

```diff
@@ -9,6 +9,7 @@ const {
 const { loadSpecTests } = require('../spec');
```

Comment:
Maybe we rename this to `command_monitoring.test.js` to match the directory name change? Or would you rather do that as part of the greater test cleanup? I'm fine either way.

Reply:
I thought about doing it, but (1) I wasn't sure if we had in progress work touching this file and what potential merge conflict issues that might cause and (2) there are a lot of references to APM throughout and as you pointed out, the greater test refactor later is likely going to change a lot of things around, so not sure it's worth doing it

Reply:
Yeah I think it's fine to wait. LGTM.